### ISSUE TITLE
Panic when passing `-p` flag on macOS

### ISSUE THREAD

**Issue by peterkos** [bensadeh/tailspin#48]
Hi!

Running the following command panics at the end of the file:
```sh
> cargo run example-logs/example2 -p
// ... the rest of the file ...
2022-09-10 02:37:29,743 [Worker-63: Building] INFO // ...
The application panicked (crashed).
Message:  called `Result::unwrap()` on an `Err` value: Custom { kind: Other, error: "task was cancelled" }
Location: src/main.rs:60
```

And on the offending line, we find ourselves with a gift! (an unwra- like u-unwrapping a gift, opening a present, etc. etc.)

https://github.com/bensadeh/tailspin/blob/c6ed6ab47064e43dc4b89bedb1816989a5fa25f7/src/main.rs#L53-L61

**Comment by bensadeh** [comment]
Thanks for reporting the issue! Glad this was easy to reproduce 😅 .

**Comment by bensadeh** [comment]
Fixed in e28051c.
